vumi_twilio_api/twilio_api.py

```diff
@@ -62,10 +79,28 @@ def format_xml(dct, root=None):
 
     @staticmethod
     def format_json(dct):
-        return json.dumps(dct)
+        if dct.get('Call'):
+            dct = dct['Call']
+        if dct.get('Version'):
+            dct = dct['Version']
```

Comment:
It's for the root of the API: `https://api.twilio.com/2010-04-01/`. I added the root in a previous PR to test that the server was working, and since we have the `Calls` resource we don't actually need it anymore. Should we leave it in or take out the root resource completely?